packages/terra-markdown/src/Markdown.scss

```diff
@@ -1,11 +1,114 @@
-.markdown-body {
-  margin-bottom: 1.25rem;
 
-  code {
-    background-color: #ebebef;
+
+:local(.markdown) {
```

Comment:
So this will locally scope `.markdown` and leave the internal selectors global?

Reply:
Yep, that's what happening.